CMakeLists.txt

```diff
@@ -155,9 +155,14 @@ else()
   endif()
 endif()
 
+find_package(Python3 3.8 REQUIRED COMPONENTS Interpreter)
+set(PYTHON_COMMAND ${Python3_EXECUTABLE})
```

Comment:
This makes us dependent on python, even if we're not running tests/bench, no? I wouldn't expect python to be a hard requirement for building libbitcoinkernel, for example.

Reply:
The `REQUIRED` option has been [removed](https://github.com/hebasto/bitcoin/pull/15#issuecomment-1546725142) for now. Suggesting to postpone the further discussion until all places where python is required will be introduced.